<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2071</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableInfo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>971</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>325</number>
    </attribute>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Difference</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Result</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>260</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>140</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>30</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Enter date in format dd.mm.yyyy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>340</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Previos day</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IsLeapButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>420</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Leap ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BirthdayButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>500</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Days till bitrthday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DurationButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>580</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>860</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nowButton">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeDateButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>660</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Change date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteDateButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>740</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>220</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxChange">
    <property name="geometry">
     <rect>
      <x>1701</x>
      <y>668</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDelete">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>740</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2071</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
